CMakeLists.txt

```diff
@@ -30,13 +45,34 @@ if(NOT CMAKE_C_LINK_PIE_SUPPORTED)
                   "If PIE is enabled by default, test auto discovery might fail.")
 endif()
 
-find_program(SPHINX_BIN sphinx-build REQUIRED)
-find_program(LCOV_BIN lcov REQUIRED)
-find_program(GENHTML_BIN genhtml REQUIRED)
-find_program(CLANG_TIDY_BIN NAMES clang-tidy-18 clang-tidy REQUIRED)
-find_program(CLANG_FORMAT_BIN NAMES clang-format-18 clang-format REQUIRED)
+if(BUILD_DOCS)
+    find_program(SPHINX_BIN sphinx-build REQUIRED)
+    find_program(LCOV_BIN lcov REQUIRED)
+    find_program(GENHTML_BIN genhtml REQUIRED)
+    find_program(CLANG_TIDY_BIN NAMES clang-tidy-18 clang-tidy REQUIRED)
+    find_program(CLANG_FORMAT_BIN NAMES clang-format-18 clang-format REQUIRED)
+endif()
+
+# Replace mandatory REQUIRED flag
+find_program(BPFTOOL_BIN bpftool)
+
+# Add fallback paths
+if(NOT BPFTOOL_BIN)
+    find_program(BPFTOOL_BIN bpftool
+        PATHS
+        ${CMAKE_SYSROOT}/usr/sbin
+        ${CMAKE_SYSROOT}/sbin
+        /usr/sbin
+        /sbin
+    )
+endif()
+
+# Make it optional with warning
+if(NOT BPFTOOL_BIN)
+    message(WARNING "bpftool not found - some features may be disabled")
+    set(BPFTOOL_BIN "bpftool") # Set default value
+endif()
```

Comment:
Move this to `tests/bpf/CMakeLists.txt`, where `bpftool` is actually used. It's not ideal, but better than the current situation and probably sufficient for your use case.